### DISCUSSION TITLE
Use different naming scheme for `--font-weight-*` and `--radius-*`?

### DISCUSSION THREAD

**Discussion by xNyfPtx** [tailwindlabs/tailwindcss#15209] [Ideas]
```
--font-weight-thin: 100;
--font-weight-extralight: 200;
--font-weight-light: 300;
--font-weight-normal: 400;
--font-weight-medium: 500;
--font-weight-semibold: 600;
--font-weight-bold: 700;
--font-weight-extrabold: 800;
--font-weight-black: 900;
```
```
--radius-xs: 0.125rem;
--radius-sm: 0.25rem;
--radius-md: 0.375rem;
--radius-lg: 0.5rem;
--radius-xl: 0.75rem;
--radius-2xl: 1rem;
--radius-3xl: 1.5rem;
--radius-4xl: 2rem;
```

Why not just use `rounded-*` instead of `radius-*` and `font-*` instead of `font-weight-*` to make it correspond to the classes?

> **Shameless self-promo**: [TailwindCSS classes as variables / design tokens](https://github.com/geomydas/twprops). It could possibly help as I keep most of the tailwind classes as variables which could better improve the css variables from tailwindcss v4.

**Comment by adamwathan** [comment]
We already use `font-*` for the font-family utilities and each variable namespace needs to be unique otherwise things are ambiguous.

For `--radius-*` it's because we're considering renaming those utilities and including `rounded-*` just for backward compatibility: 

https://github.com/tailwindlabs/tailwindcss/pull/14912
